### PR TITLE
feat: add syslog format for a10networks

### PR DESCRIPTION
**Pull request by ikheifets-splunk**

### CODE REVIEW

File: package/etc/conf.d/conflib/syslog/app-syslog-a10.conf [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+                vendor('a10networks')
+                product('vThunder')
+            );
+        };
+    };
+};
+
+application app-syslog-a10[sc4s-syslog-pgm] {
+    filter {
+        program('a10logd' type(string) flags(prefix));
```
Comment (ikheifets-splunk) [inline package/etc/conf.d/conflib/syslog/app-syslog-a10.conf:17]: @rjha-splunk @mstopa-splunk Is it okay or `program('a10logd' type(string) flags(prefix)) and message('[audit log]'  type(string)  flags(prefix))` would be better? I just really worry about that `a10logd` might be suite for different devices for that vendor

File: docs/sources/vendor/a10networks/vthunder.md [hunk 2]
```diff
@@ -16,9 +16,10 @@
 
 ## Sourcetypes
 
-| sourcetype               | notes                                                                                         |
-|--------------------------|-----------------------------------------------------------------------------------------------|
-| a10networks:vThunder:cef | None                                                                                          |
+| sourcetype                  | notes                                                                                      |
+|-----------------------------|--------------------------------------------------------------------------------------------|
+| a10networks:vThunder:cef    | CEF                                                                                        |
+| a10networks:vThunder:syslog | Syslog                                                                                     |
```
Comment (ikheifets-splunk) [inline docs/sources/vendor/a10networks/vthunder.md:22]: Before that customer sent us pcap only with CEF logs, it's seems that device producing logs in 2 formats

File: package/etc/conf.d/conflib/syslog/app-syslog-a10.conf (hunk 1 above)
Comment (mstopa-splunk) [reply package/etc/conf.d/conflib/syslog/app-syslog-a10.conf:17]: `a10logd` will be sufficient

**Review by mstopa-splunk — APPROVED** [review]

**Pull request merged**